frontend/talentsearch/src/js/components/Home/partials/Featured/Featured.tsx

```diff
@@ -4,54 +4,86 @@ import { useIntl } from "react-intl";
 import { imageUrl } from "@common/helpers/router";
 
 import Heading from "@common/components/Heading";
+import { getLocale } from "@common/helpers/localize";
 import TALENTSEARCH_APP_DIR from "../../../../talentSearchConstants";
 
 import Block from "./Block";
 
 // Create the page component
 const Featured = () => {
   const intl = useIntl();
+  const locale = getLocale(intl);
+
+  const iapEmail = {
+    subject: encodeURIComponent(
+      "I'm interested in hiring Indigenous IT apprentices for my team",
+    ),
+    body: encodeURIComponent(
+      "I discovered the Indigenous Apprenticeship Program on talent.canada.ca and I’d like to learn more about how I can hire apprentices to my team.",
+    ),
+  };
+
   // This array is just a temporary data object representing the content required by the feature blocks. This data will need to be migrated to wherever makes sense, and we'll also need dynamic routes and translated strings
   const featured = [
     {
-      key: "pride",
+      key: "digital-ambition",
       title: intl.formatMessage({
-        defaultMessage: "Pride in tech",
-        id: "muwFhL",
-        description: "Title for the pride in tech featured item",
+        defaultMessage: "The Digital Ambition",
+        id: "tTuBmE",
+        description: "Title for the digital ambition featured item",
+      }),
+      summary: intl.formatMessage({
+        defaultMessage:
+          'The Digital Ambition outlines the Government of Canada\'s commitment to create modern, accessible digital services. Achieving these priorities will result in a government that provides improved "digital-first," user-centred, and barrier-free services and programs.',
```

Comment:
Any reason why you use dumb quotes here but smart quotes in other strings?

Reply:
I honestly just didn't realize. Did we ever decide on dumb vs smart? I can go through the entire homepage and make sure all strings use one or the other.

Reply:
Okay, went through and removed all smart quotes since that seemed to be the most common.

[clean up copy (whitespace, quotes)](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4442/commits/a739b51e2dc86a2cf4f4b20de407ec2995705665)